src/Events/Event.php

```diff
@@ -0,0 +1,208 @@
+        $this->output->info("You lost $creditsLost credits");
+        $player->payCredits($creditsLost);
+    }
+
+    private function gamblingEvent(Pilot $player): void
+    {
+        $this->output->write("");
+        $this->output->info("You spotted a few people playing cards. You approach their table.");
+        $options = ["Yes" => "Yes", "No" => "No"];
+        $result = $this->output->getCli()->radio("Do you want to play some cards?", $options)->prompt();
+
+        $this->output->write("");
+        if ($result === "Yes") {
+            $this->output->write("You decide to gamble some money");
+            $this->output->write("You have " . $player->getCredits() . " credits.");
+            $credits = $this->output->input("How much do you want to gamble?", $player->getCredits());
+            $win = rand(1, 2);
+            if ($win == 1) {
+                $this->output->write("");
+                $this->output->info("You won! You earned $credits credits.");
+                $player->earnCredits($credits);
+            }
+            if ($win == 2) {
+                $this->output->write("");
+                $this->output->info("You lost! You lost $credits credits.");
+                $player->payCredits($credits);
+            }
+        }
+        if ($result === "No") {
+            $this->output->write("");
+            $this->output->write("You decide to leave the players alone. You are above gambling.");
+        }
+    }
+
+    private function racingEvent(Pilot $player): void
+    {
+        $this->output->write("");
+        $this->output->info("You spotted an ad for pod racing.");
+        $this->output->info("You decide to compete in the race.");
+        $bestPilot = rand(1, 10);
+
+        if ($player->getSkill() > $bestPilot) {
+            $this->output->write("");
+            $this->output->info("You won the race by beating all other racers.");
+            $this->output->info("You earned 1000 credits.");
+            $player->earnCredits(1000);
+        }
+        if ($player->getSkill() < $bestPilot) {
+            $this->output->write("");
+            $this->output->info("You lost the race.");
+            $this->output->info("You didn't lose anything other than time.");
+        }
+    }
+
+    private function bountyHunterEvent(Pilot $player): void
+    {
+        $bounty = $player->calculateBounty();
+        $this->output->write("You were approached by a bounty hunter");
+        $this->output->write("He said that you have a bounty of $bounty credits on your head but for a price he can pretend you two never met.");
+
+
+        if ($player->getCredits() < $bounty) {
+            $this->output->info("That's too bad partner...");
+            $this->output->write("The next thing you heard was a shot from his blaster.");
+            $this->output->info("You've met your demise.");
+            exit(0);
```

Comment:
Może zamisat pustego `exit` warto by było zrobić klasę, która by zarządzała obecnym stanem gry?

Reply:
Podchodziłem do tego trochę jak do gry z salonu gier, gdzie przegrana oznacza to że trzeba zacząć na nowo dlatego skorzystałem z `exit`